core/client/src/main/java/alluxio/client/netty/NettyClient.java

```diff
@@ -56,31 +59,29 @@
       Configuration.getInt(PropertyKey.USER_NETWORK_NETTY_WORKER_THREADS), "netty-client-worker-%d",
       true);
 
-  /** The maximum number of milliseconds to wait for a response from the server. */
-  public static final long TIMEOUT_MS =
```

Comment:
This was not used after non-packet streaming code is removed. It is now defined in NettyPacketReader/Writer.